event/tests/test_event_list_ui.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+from ..models import Event
+
+
+@pytest.mark.django_db
+class TestEventList:
+    def test_event_list(self, client, user1):
+        events = Event.manager.all()
+        client.force_login(user1.user)
```

Comment:
maybe instead of repeating these lines in each test 
you can create fixtures something like:

@pytest.fixture
def all_events():
    return Event.manager.all()

@pytest.fixture
def logged_in_client(client, user1):
    client.force_login(user1.user)
    return client

```suggestion
    def test_event_list(self, logged_in_client, all_events, user1):
```